### PR TITLE
planner: introduce 2 new variables 

### PR DESCRIPTION
**Pull request by qw4990**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -1008,6 +1008,20 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 默认值：`OFF`
 - 这个变量用来设置优化器是否执行聚合函数下推到 Join，Projection 和 UnionAll 之前的优化操作。当查询中聚合操作执行很慢时，可以尝试设置该变量为 ON。
 
+### `tidb_opt_limit_push_down_threshold`
+
+- 作用域：SESSION | GLOBAL
+- 默认值：`100`
+- 范围：`[0, 2147483647]`
+- 这个变量用来设置 Limit 和 TopN 下推的阈值。
+- 如果 Limit 或者 TopN 的取值个数小于这个阈值，则会被强制下推到 TiKV，该变量可以解决部分由于估算误差导致 Limit 或者 TopN 无法被下推的问题。
```
Comment (time-and-fate) [inline system-variables.md:1017]:
```suggestion
- 如果 Limit 或者 TopN 的取值个数小于等于这个阈值，则会被强制下推到 TiKV，该变量可以解决部分由于估算误差导致 Limit 或者 TopN 无法被下推的问题。
```

**Review by time-and-fate — APPROVED** [review]

File: system-variables.md [hunk 2]
```diff
@@ -1008,6 +1008,20 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 默认值：`OFF`
 - 这个变量用来设置优化器是否执行聚合函数下推到 Join，Projection 和 UnionAll 之前的优化操作。当查询中聚合操作执行很慢时，可以尝试设置该变量为 ON。
 
+### `tidb_opt_limit_push_down_threshold`
+
+- 作用域：SESSION | GLOBAL
+- 默认值：`100`
+- 范围：`[0, 2147483647]`
+- 这个变量用来设置 Limit 和 TopN 下推的阈值。
+- 如果 Limit 或者 TopN 的取值个数小于等于这个阈值，则会被强制下推到 TiKV，该变量可以解决部分由于估算误差导致 Limit 或者 TopN 无法被下推的问题。
```
Comment (qiancai) [inline system-variables.md:1017]:
```suggestion
- 如果 Limit 或者 TopN 的取值个数小于等于这个阈值，则 Limit 和 Sort 算子会被强制下推到 TiKV，该变量可以解决部分由于估算误差导致 Limit 或者 TopN 无法被下推的问题。
```

Comment (qiancai) [reply system-variables.md:1017]: @qw4990 PTAL

File: system-variables.md [hunk 3]
```diff
@@ -1008,6 +1008,20 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 默认值：`OFF`
 - 这个变量用来设置优化器是否执行聚合函数下推到 Join，Projection 和 UnionAll 之前的优化操作。当查询中聚合操作执行很慢时，可以尝试设置该变量为 ON。
 
+### `tidb_opt_limit_push_down_threshold`
+
+- 作用域：SESSION | GLOBAL
+- 默认值：`100`
+- 范围：`[0, 2147483647]`
+- 这个变量用来设置 Limit 和 TopN 下推的阈值。
```
Comment (TomShawn) [inline system-variables.md:1016]:
```suggestion
- 这个变量用来设置将 Limit 和 TopN 算子下推到 TiKV 的阈值。
```

File: system-variables.md [hunk 4]
```diff
@@ -1008,6 +1008,20 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
+- 如果 Limit 或者 TopN 的取值个数小于等于这个阈值，则 Limit 和 Sort 算子会被强制下推到 TiKV，该变量可以解决部分由于估算误差导致 Limit 或者 TopN 无法被下推的问题。
```
Comment (TomShawn) [inline system-variables.md:1017]:
```suggestion
- 如果 Limit 或者 TopN 的取值小于等于这个阈值，则 Limit 和 Sort 算子会被强制下推到 TiKV。该变量可以解决部分由于估算误差导致 Limit 或者 TopN 无法被下推的问题。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**